client/src/panels/panel_declarations/people/employee_age.js

```diff
@@ -166,28 +166,30 @@ export const declare_employee_age_panel = () => declare_panel({
             <TM k={level+"_employee_age_text"} args={info} />
           </Col>
           { !window.is_a11y_mode &&
-            <Col size={12} isGraph extraClasses="zero-padding"> 
-              <TabbedContent
-                tab_keys={["age_group", "avgage"]}
-                tab_labels={{
-                  age_group: text_maker("age_group"),
-                  avgage: text_maker("avgage"),
-                }}
-                tab_pane_contents={{
-                  age_group: (
-                    <div id={"emp_age_tab_pane"}>
-                      <LineBarToggleGraph {...age_group_options} />
-                      <div className='clearfix'></div>
-                    </div>
-                  ), 
-                  avgage: (
-                    <div id={"emp_age_tab_pane"}>
-                      <LineBarToggleGraph {...avg_age_options} />
-                      <div className='clearfix'></div>
-                    </div>
-                  ),
-                }}
-              />
+            <Col size={12} isGraph extraClasses="zero-padding">
+              <div aria-hidden="true">
```

Comment:
This is where the `aria-hidden="true"` from `TabbedContent` actually belonged.

Reply:
hides tabbed content correctly, makes sense
👍